predicators/ground_truth_models/kitchen/nsrts.py

```diff
@@ -70,9 +69,11 @@ def push_sampler(state: State, goal: Set[GroundAtom],
             y = state.get(gripper, "y")
             z = state.get(gripper, "z")
             if obj.name == 'knob3':
-                return np.array([x + 0.25, y, z], dtype=np.float32)
+                return np.array([x + 1.0, y, z], dtype=np.float32)
             if obj.name == 'kettle':
-                return np.array([x, y + 5.0, z - 0.3], dtype=np.float32)
+                rand_dx = _rng.uniform(0.0, 1.0)
```

Comment:
We are only adding noise to the x parameter when pushing the kettle? I guess for now it is ok, but eventually learning on top of one more generic sampler is what we want.

Reply:
yeah this was just enough to get it to like 30% base success